cpp/INIReader.cpp

```diff
@@ -56,6 +56,16 @@ long INIReader::GetInteger(const string& section, const string& name, long defau
     return end > value ? n : default_value;
 }
 
+unsigned long INIReader::GetUnsignedInteger(const string& section, const string& name, unsigned long default_value) const
```

Comment:
Let's call this `GetUnsigned` for brevity (like with C / C++ itself, the "integer" is kind of obvious).
```suggestion
unsigned long INIReader::GetUnsigned(const string& section, const string& name, unsigned long default_value) const
```